spyder/plugins/editor/confpage.py

```diff
@@ -264,112 +321,61 @@ def enable_tabwidth_spin(index):
         autosave_group.setLayout(autosave_layout)
 
         # -- Docstring
-        docstring_group = QGroupBox(_('Docstring type'))
+        docstring_group = QGroupBox(_('Docstring style'))
 
         numpy_url = "<a href='{}'>Numpy</a>".format(NUMPYDOC)
         googledoc_url = "<a href='{}'>Google</a>".format(GOOGLEDOC)
         sphinx_url = "<a href='{}'>Sphinx</a>".format(SPHINXDOC)
         docstring_label = QLabel(
-            _("Here you can select the type of docstrings ({}, {} or {}) you "
-              "want the editor to automatically introduce when pressing "
-              "<tt>{}</tt> after a function, method or class "
-              "declaration.").format(
-                  numpy_url, googledoc_url, sphinx_url, DOCSTRING_SHORTCUT)
+            _("Select the style of docstrings ({numpy}, {google} or {sphinx}) "
+              "to generate when pressing <kbd>{shortcut}</kbd> after a "
+              "function, method or class declaration").format(
+                  numpy=numpy_url,
+                  google=googledoc_url,
+                  sphinx=sphinx_url,
+                  shortcut=DOCSTRING_SHORTCUT,
+              ),
         )
         docstring_label.setOpenExternalLinks(True)
         docstring_label.setWordWrap(True)
 
-        docstring_combo_choices = ((_("Numpy"), 'Numpydoc'),
-                                   (_("Google"), 'Googledoc'),
-                                   (_("Sphinx"), 'Sphinxdoc'),)
+        docstring_combo_choices = (
+            (_("Numpy"), 'Numpydoc'),
+            (_("Google"), 'Googledoc'),
+            (_("Sphinx"), 'Sphinxdoc'),
```

Comment:
```suggestion
            ("Numpy", 'Numpydoc'),
            ("Google", 'Googledoc'),
            ("Sphinx", 'Sphinxdoc'),
```

No need to translate these strings (I know it was like that before but we can take the chance to fix it).

Reply:
Applied ✅